crates/hc_sandbox/tests/cli.rs

```diff
@@ -602,12 +631,44 @@ async fn get_created_conductor_config(process: &mut Child) -> ConductorConfig {
             let config = read_config(PathBuf::from_str(config_root_path).unwrap().into())
                 .unwrap()
                 .unwrap();
+
+            // On windows, this task stays alive and holds the stdout pipe open
+            // so that the tests don't finish.
+            #[cfg(not(windows))]
+            tokio::task::spawn(async move {
+                while let Ok(Some(line)) = lines.next_line().await {
+                    println!("@@@-{line}-@@@");
+                }
+            });
+
             return config;
         }
     }
     panic!("getting created conductor config failed");
 }
 
+async fn shutdown_sandbox(mut child: Child) {
+    #[cfg(unix)]
+    {
+        use nix::sys::signal::Signal;
+        use nix::unistd::Pid;
+
+        let pid = child.id().expect("Failed to get PID");
+        nix::sys::signal::kill(Pid::from_raw(pid as i32), Signal::SIGINT)
+            .expect("Failed to send SIGINT");
+
+        let exit_code = child.wait().await.unwrap();
+        assert!(exit_code.success());
+    }
+
+    #[cfg(not(unix))]
+    {
+        // Best effort to shut down for platforms that don't support sending signals in a
+        // simple way.
+        child.kill().await.unwrap();
```

Comment:
This actually seems to work okay on Windows, so I guess the process handing is just different there